spec/Vehicle/Vehicle.vspec

```diff
@@ -184,16 +184,43 @@ Speed:
   description: Vehicle speed.
 
 TravelledDistance:
+  deprecation: V3.1 moved to Vehicle.TraveledDistance
```

Comment:
Today we are mixing British and American English. While poking around in this area, do we think it is a good idea to rename TravelledDistance to TraveledDistance (using our deprecated-methodology, i.e. keeping old signal until v4.0)